src/lib/csrf.ts

```diff
@@ -1,7 +1,8 @@
 import { csrfSync } from "csrf-sync";
+import { Request } from "express";
 
 const { csrfSynchronisedProtection } = csrfSync({
-  getTokenFromRequest: (req) => {
+  getTokenFromRequest: (req: Request) => {
```

Comment:
(this argument should be inferred from the csrf-sync types - but you can specify it if you like)